src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsController.java

```diff
@@ -112,15 +111,15 @@
         log.info("Received request to send email for 'Consent Order Available' for Case ID: {}", callbackRequest.getCaseDetails().getId());
         validateCaseData(callbackRequest);
         Map<String, Object> caseData = callbackRequest.getCaseDetails().getData();
-        if (isSolicitorAgreedToReceiveEmails(caseData, SOLICITOR_AGREE_TO_RECEIVE_EMAILS)) {
+        if (hasSolicitorAgreedToReceiveEmails(caseData)) {
             log.info("Sending email notification to Solicitor for 'Consent Order Available'");
             notificationService.sendConsentOrderAvailableEmail(callbackRequest);
         }
         return ResponseEntity.ok(AboutToStartOrSubmitCallbackResponse.builder().data(caseData).build());
     }
 
-    private boolean isSolicitorAgreedToReceiveEmails(Map<String, Object> mapOfCaseData, String solicitorAgreeToReceiveEmails) {
+    private boolean hasApplicantSolicitorAgreedToReceiveEmails(Map<String, Object> mapOfCaseData) {
```

Comment:
why not hasSolicitorAgreedToReceiveEmails from `commonConditions`?

Reply:
This was a mistake that i had forgotten to remove. updating now